installer/scripts/omsadmin.sh

```diff
@@ -787,6 +804,36 @@ show_workspace_status()
     fi
 }
 
+show_omsagent_proc_limit()
+{
+    llimit=-1
+    resultcode=0
+
+    count=`cat $PROC_LIMIT_CONF | grep -E "$LIMIT_LINE_REGEX" | wc -l`
+
+    if [ "$count" -eq "0" ]; then
+        log_info "INFO:  OMS Agent process limit NOT Presently Set."
+    elif [ $count -eq 1 ]; then
+        llimit=`cat $PROC_LIMIT_CONF | grep -E "$LIMIT_LINE_REGEX" | awk '{print $4}'`
+    else
+        log_warning "$count $AGENT_USER entries found in $PROC_LIMIT_CONF.  This may yield problems.  Last entry should apply."
+        log_warning ">>Last entry presumed to apply<<"
+        grep -E "$LIMIT_LINE_REGEX" $PROC_LIMIT_CONF 1>&2
+        log_warning "End of the $count duplicate entry WARNING Message"
+        llimit=`grep -E "$LIMIT_LINE_REGEX" $PROC_LIMIT_CONF | tail -1 | awk '{print $4}'`
+    fi
+
+    if [ "$llimit" -eq -1 ]; then
+        log_info "$AGENT_USER proc count unlimited."
```

Comment:
There is a case, must needs be by hand, that the user could intentionally code the configuration with an explicit -1.  This would pick that up, but the other would not.

Reply:
Also, the other tells the state of the maintenance, and this tells the behavior.